client/Encounter/Encounter.test.ts

```diff
@@ -53,4 +53,12 @@ describe("Encounter", () => {
             expect(encounter.Combatants()[0]).toBe(encounter.ActiveCombatant());
         }
     });
+
+    test("Encounter turn timer stops when encounter ends", () => {
+        encounter.AddCombatantFromStatBlock({ ...StatBlock.Default(), HP: { Value: 10, Notes: "" }, Player: "player" });
+        encounter.StartEncounter();
+        (<any>encounter.TurnTimer).elapsedSeconds(10);
```

Comment:
Instead of using private members of this class to test, you should use public members, which will avoid the need for this casting. In this case that involved calling TurnTimer.Start(), then using [Jest's Timer mocks](https://jestjs.io/docs/en/timer-mocks.html#advance-timers-by-time) to advance the actual runtime 10 seconds.